packages/tools/test/LengthTool_test.js

```diff
@@ -586,13 +587,15 @@ describe('LengthTool:', () => {
           pageX: pageX3,
           pageY: pageY3,
         });
-        element.dispatchEvent(evt);
-
         // Just grab and don't really move it
-        evt = new MouseEvent('mouseup');
-
-        addEventListenerForAnnotationRendered();
-        document.dispatchEvent(evt);
+        const mouseUpEvt = new MouseEvent('mouseup');
+
+        doMouseDownAndUp(
+          element,
+          evt,
```

Comment:
I know evt, is mouseDown, but renaming the variable to `mouseDownEvt` would match the `mouseUpEvt` nicelu

Reply:
Sure. I will go through the various unit test files and make updates.